utils/nctl/sh/scenarios/common/itst.sh

```diff
@@ -208,3 +208,18 @@ function assert_node_proposed() {
     log "node-$NODE_ID created a block!"
     log "$OUTPUT"
 }
+
+# Checks logs for nodes 1-10 for equivocators
+function assert_no_equivocators_logs() {
+    local LOGS
+    log_step "Looking for equivocators in logs..."
+    for i in {1..10}; do
```

Comment:
I think there's an `nctl` function that returns the number of nodes created. That would be more resilient.

Reply:
added in https://github.com/casper-network/casper-node/pull/1374/commits/27419f591aebc07149358edad135ea558f78d81c